client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap');
+
+.PinkHeader {
+    background-image: url('./cotripper_banner2-01.png');
+    background-repeat:no-repeat;
+    background-size: cover;
+    width: 1440px;
+    height: 231px;
+    text-align: center;
+    padding-top: 3px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+h3{
+    font-family: 'Reem Kufi', sans-serif;
+    color: #FAFAFA;
+    font-size: 48px;
+    line-height: 72px;
+    text-align: center;
+    margin: 25px 0 0 0;
+}
+
+.box2{   
+    margin: 0 auto;
+    width: 356px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+}
+
+
+.container-1 input#search {
+    width: 356px;
+    height: 48px;
+    background-color: #FAFAFA;
```

Comment:
Let's use the css variables for colors here. For instance, that background color would be:
background-color: var(--white);
You can refer to App.css for any variables.
Keep rockin!

Reply:
+1